stories/form-grid/form-grid.stories.js

```diff
@@ -1371,3 +1371,219 @@ fourSevenOneCompactLayout.parameters = {
+          <input class="fd-input" type="text" id="input-1-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+
+    <div class="fd-col fd-col-md--6 fd-col-lg--6 fd-col-xl--6 fd-col--wrap">
+      <div class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+          <label class="fd-form-label" for="input-1a-name">Name:</label>
+        </div>
+        <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+          <input class="fd-input" type="text" id="input-1a-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<h1>XL2-L2-M1-S1</h1>
+<div class="fd-container fd-form-layout-grid-container">
+  <div class="fd-row"> 
+    <div class="fd-col fd-col-md--12 fd-col-lg--6 fd-col-xl--6 fd-col--wrap">
+      <div class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+          <label class="fd-form-label" for="input-2-name">Name:</label>
+        </div>
+        <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+          <input class="fd-input" type="text" id="input-2-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+
+    <div class="fd-col fd-col-md--12 fd-col-lg--6 fd-col-xl--6 fd-col--wrap">
+      <div class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+          <label class="fd-form-label" for="input-2a-name">Name:</label>
+        </div>
+        <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+          <input class="fd-input" type="text" id="input-2a-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<h1>XL2-L1-M1-S1</h1>
+<div class="fd-container fd-form-layout-grid-container">
+  <div class="fd-row"> 
+    <div class="fd-col fd-col-md--12 fd-col-xl--6 fd-col--wrap">
+      <div class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+          <label class="fd-form-label" for="input-3-name">Name:</label>
+        </div>
+        <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+          <input class="fd-input" type="text" id="input-3-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+
+    <div class="fd-col fd-col-md--12 fd-col-xl--6 fd-col--wrap">
+      <div class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+          <label class="fd-form-label" for="input-3a-name">Name:</label>
+        </div>
+        <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+          <input class="fd-input" type="text" id="input-3a-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+`;
+
+columnRecommended.parameters = {
```

Comment:
My suggestion would be to make this your main header:

"**Use of Columns**
When working with form groups, it's always best to use the recommended number of columns to make the most of your screen space. That way, users aren't prompted to scroll down because of unused white space, and the form is visually balanced between the left and right side of the screen."

Then place these headers underneath:
**Recommended
Possible
Not recommended**

Reply:
I put the first part at the top while we figure out a custom way to include sub headers and sub paragraphs in the middle of our stories while not associating them to one story. I've also made the description clearer for the sizing.